src/main/java/io/codearte/jfairy/producer/net/NetworkProducer.java

```diff
@@ -18,4 +18,17 @@ public String ipAddress() {
 		return ipNumberProducer.generate();
 	}
 
+	/**
+	 * Add a simple url generator
+	 * @return
+	 */
+	public String url() {
```

Comment:
In some cases I need to generate random url for each user, and the original company urls is not enough. I assumed this is a very common demand and I couldn't find a generator in JFairy except company url. And as what I mentioned above, I believe that each company with a fixed url is better.